app/src/main/java/com/practicum/playlistmaker/TrackAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.practicum.playlistmaker
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+
+class TrackAdapter (val trackList: MutableList<Track>): RecyclerView.Adapter<TrackViewHolder>() {
+
+    private var filteredTrackList: MutableList<Track> = trackList.toMutableList()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): TrackViewHolder {
+        val itemView = LayoutInflater.from(parent.context).inflate(R.layout.item_track, parent, false)
+        return TrackViewHolder(itemView)
+    }
+
+    override fun onBindViewHolder(holder: TrackViewHolder, position: Int) {
+        val track = filteredTrackList[position]
+        holder.trackName.text = track.trackName
+        holder.artistName.text = track.artistName
+        holder.trackTime.text = track.trackTime
+        Glide.with(holder.trackImage.context).load(track.artworkUrl100).into(holder.trackImage)
```

Comment:
Здесь нужно просто вызвать функцию `bind` из вьюхолдера:
```suggestion
        val track = filteredTrackList[position]
        holder.bind(track)
```